dhis-2/dhis-api/src/main/java/org/hisp/dhis/appmanager/App.java

```diff
@@ -95,9 +95,12 @@
 
     private Set<String> authorities = new HashSet<>();
 
-    private AppStatus appState = AppStatus.OK;
+    private boolean coreApp = false;
 
-    private boolean isBundledApp = false;
```

Comment:
I think a better name here is just `bundled`. The "is" prefix is normally avoided in booleans, as "is" the regular prefix for get methods. The "app" part is implicit since the object is called App.

Reply:
@larshelge makes sense.  This line is actually removed, now we only have the getter and setter (previously the variable wasn't set until initialization with a contextPath, which caused problems when trying to evaluate the app at installation time).  Should I update the getter [here](https://github.com/dhis2/dhis2-core/pull/6111/files#diff-787d95701935d2074664ae90d6873da0R138-R142) or leave it as before?

Reply:
Ah okay, right. Will be break any clients by making that change? If not, go for it.

Reply:
I'll have to update all the references to it (there are a few) but doable